.ado/xcconfig/publish_overrides_ios_device.xcconfig

```diff
@@ -2,4 +2,3 @@
 
 // Explicitly build arm64e for iOS device builds in case our clients need it
 ARCHS = $(ARCHS_STANDARD) arm64e
-ENABLE_ADDRESS_SANITIZER = NO
```

Comment:
no need for this file change?

Reply:
This file includes the other one that has the flag to disable it, so it's redundant:

`#include "publish_overrides.xcconfig"`